src/main/java/com/conveyal/gtfs/loader/DateListField.java

```diff
@@ -18,19 +18,22 @@ public DateListField(String name, Requirement requirement) {
     }
 
     @Override
-    public String validateAndConvert(String original) {
-        // FIXME
+    public ValidateFieldResult<String> validateAndConvert(String original) {
+        // FIXME: This function is currently only used during feed loading. Because the DateListField only exists for the
```

Comment:
I don't like seeing FIXMEs in comments. Can we just remove the FIXME from the comment and call it good for now? If not, how would we go about fixing this?

Reply:
Can you explain why you want fix-me comments to be removed? I have added many hundreds of such comments all over our code bases, so if there is a decision to remove them or not do this it will make a big difference for me. While working on something, various problems and concerns are identified, and it's a real loss for those to be forgotten. It's impossible to branch off development every time a new idea comes up, and it's often better to let ideas mature and confront them with later more complete stages of implementation. Often those ideas or problems need to be seen by the next person working on the code.

Reply:
The above is a general response about fix-me comments. In this specific case though, I agree that this should just be fixed. Where a function in an interface is not needed on a special-purpose subclass, in my opinion the proper thing to do is to throw an "operation not supported" exception, and include a comment much like this one that explains why the operation is not supported.

Reply:
In the general sense, FIXME to me seems to indicate something is wrong and must be corrected. I'd prefer using TODO in those cases when an idea should be fleshed out a little better.

Reply:
Ah, if it's just a question of not exaggerating severity, as well as fixing small things now instead of later, then I see what you mean.